test/unittests/eof_validation_test.cpp

```diff
@@ -815,96 +803,87 @@ TEST_F(eof_validation, EOF1_dataloadn_truncated)
 TEST_F(eof_validation, dataloadn)
 {
     // DATALOADN{0}
-    add_test_case(
-        "EF0001 010004 0200010005 040020 00 00800001 D100005000 "
-        "0000000000000000111111111111111122222222222222223333333333333333",
+    add_test_case(eof_bytecode(bytecode{} + OP_DATALOADN + "0000" + OP_POP + OP_STOP, 1)
```

Comment:
here and in several cases below
```suggestion
    add_test_case(eof_bytecode(bytecode{OP_DATALOADN} + "0000" + OP_POP + OP_STOP, 1)
```
or you could instead add `dataloadn(uint16_t)` utility to `bytecode.hpp`